src/main/java/org/cms/bloggyblog/model/transfer/Entry.java

```diff
@@ -0,0 +1,20 @@
+package org.cms.bloggyblog.model.transfer;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.cms.bloggyblog.model.entity.User;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@AllArgsConstructor
```

Comment:
Do you need all of `@Data` `@AllArgs..` @NoArgs...` if you aren't using an `@Entity`?